example/ck_tile/01_fmha/fmha_fwd.hpp

```diff
@@ -291,12 +304,20 @@ struct fmha_fwd_args
     ck_tile::index_t batch_stride_o;
     ck_tile::index_t mask_y;
     ck_tile::index_t mask_x;
+    typename ElementFunctions::QElementFunction q_element_func;
```

Comment:
Can you just keep the whole object here?
```c++
ElementFunctions element_funcs;
```

Reply:
As discussed, only P and o element function now, let keep unchange